scripts/require-sync/go.mod

```diff
@@ -0,0 +1,136 @@
+module require-sync
```

Comment:
can you run `go mod tidy`? i am seeing a few updates on my side and looks like `go.work.sum` file is also generated

Reply:
Done, should have done that after the rebase.